source/blood/src/config.cpp

```diff
@@ -322,6 +323,7 @@ void CONFIG_SetDefaults(void)
     MouseBias       = 0;
     MouseDeadZone   = 0;
     MusicToggle     = 1;
+    MusicRestartsOnLoadToggle = 1;
```

Comment:
Default it to 0

Reply:
That would be different from the original game, are you sure you want to make behavioral changes by default compared the the original? Are there already things what differs from the original on purpose already?

Reply:
i'm fine with it, this is not chocolate port anyway. In long run it'll be EDuke32 equivalent for blood